### DISCUSSION TITLE
Fief React Native Support

### DISCUSSION THREAD

**Discussion by saleempirwani** [fief-dev/fief#171] [Q&A]
Hi there, 

Want to ask, can we use Fief in React Native mobile app? If Yes then how? If there is anything please let me know.

**Comment by fief-bailiff** [comment]
Hail, @saleempirwani 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to star our repository ⭐️

![star-fief](https://user-images.githubusercontent.com/1144727/192523552-dc2c31fa-ae94-42fa-9ab4-7d0ba51bb7b0.png)

Farewell!

**Comment by frankie567** [comment]
Hi @saleempirwani 👋 

We don't have an official React Native integration right now. However, the base JS library will probably work and will give you the right functions and methods to perform an OAuth authentication with Fief: https://fief-dev.github.io/fief-js/classes/index.Fief.html

You'll have to come up with your own way of storing your access token / refresh token in the app storage so you can keep the session opened.

**Comment by imvikysharma** [comment]
Hello there,

Fief is not a term commonly used in the context of React Native development. Can you please clarify what you mean by "Fief" and how you would like to use it in your mobile app?

I would like to recommend a book on Rract native/js
https://www.amazon.com/dp/B0BVRKSY6G?maas=maas_adg_A78FB1C22F0F576564CF29D2524311F0_afap_abs&ref_=aa_maas&tag=maas

**Reply by frankie567** [reply]
Please don't reply with ChatGPT answers, it doesn't help.
